client/src/Pages/DashboardProvider/index.js

```diff
@@ -21,11 +32,37 @@ import './style.css';
 const { Sider, Content } = Layout;
 const { Text } = Typography;
 const { useBreakpoint } = Grid;
+
 const DashboardProvider = () => {
   const { md } = useBreakpoint();
   const [visible, setVisible] = useState(false);
+  const [isLoading, setLoading] = useState(false);
+  const [providerDetails, setProviderDetails] = useState(null);
   const [page, setPage] = useState(<Orders />);
   const [title, setTitle] = useState('Orders');
+  const [isSwitchLoading, setSwitchLoading] = useState(false);
+  const [isSwitch, setSwitch] = useState(false);
+  const { userData } = useContext(AuthContext);
+  useEffect(() => {
+    let unmounted = true;
+    (async () => {
+      try {
+        setLoading(true);
+        const { data } = await Axios.get(`/api/v1/provider/${userData.id}`);
+        if (unmounted) {
+          setLoading(false);
+          setProviderDetails(data.data[0]);
```

Comment:
use [array destract](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment) for get ``` data.data[0] ```